src/components/views/right_panel/TimelineCard.tsx

```diff
@@ -78,6 +82,7 @@ export default class TimelineCard extends React.Component<IProps, IState> {
         this.state = {
             showReadReceipts: SettingsStore.getValue("showReadReceipts", props.room.roomId),
             layout: SettingsStore.getValue("layout"),
+            atEndOfLiveTimeline: true,
```

Comment:
is it not possible to open the right panel timeline already scrolled up? e.g a permalink

Reply:
I meant in terms of `atEndOfLiveTimeline` being false before an onScroll fires

Reply:
Okay, updated to support this.

Reply:
It still seems like if the timeline gets opened to a permalink then `atEndOfLiveTimeline` will wrongly be true until the user scrolls, causing the jump button to not show up, no?

Reply:
Ah, it seems to show the button even for permalinks. That portion is wired up to `onScroll` (instead of `onUserScroll`), so when the `ScrollPanel` navigates to the highlighted event, `onScroll` is called, and we'll set `atEndOfLiveTimeline` to `false`.